script/seed.js

```diff
@@ -501,7 +497,7 @@ async function seed() {
 
   for (let i = 1; i < 40; i++) {
     reviewArray.push(
-      Review.create({
+      await Review.create({
```

Comment:
same as comment on line 477

Reply:
@russellkerns good catch there! I went ahead and took those awaits out - thanks!